e2e/test/scenarios/dashboard-cards/dashboard-card-reproductions.cy.spec.js

```diff
@@ -542,191 +542,6 @@ describe("issue 18454", () => {
   });
 });
 
-describe("adding an additional series to a dashcard (metabase#20637)", () => {
```

Comment:
No need to port this, it's already covered by any visualizer test that combines datasets.
For reference: https://github.com/metabase/metabase/issues/20637